resources/views/components/menu/main.blade.php

```diff
@@ -140,7 +140,7 @@
     <x-dropdown-item :link="url('achievementList.php?s=4&p=2')">Easy Achievements</x-dropdown-item>
     <x-dropdown-item :link="url('achievementList.php?s=14&p=2')">Hardest Achievements</x-dropdown-item>
 </x-nav-dropdown>
-<x-nav-dropdown :title="__('Community')">
+<x-nav-dropdown :title="__('Community')" desktopHref="{{ url('forum.php') }}">
```

Comment:
I don't think anyone's yearning for the menu items to be clickable. The profile image is muscle memory from the old layout.

Reply:
I can back this out if we prefer.

Reply:
sounds good to me, wasn't sure either. profile picture should be enough for now